Source/WebCore/platform/mock/MockRealtimeVideoSource.h

```diff
@@ -122,7 +124,7 @@ class MockRealtimeVideoSource : public RealtimeVideoCaptureSource, private Orien
     Color m_fillColor { Color::black };
     MockMediaDevice m_device;
     RefPtr<VideoPreset> m_preset;
-    VideoFrame::Rotation m_deviceOrientation { VideoFrame::Rotation::None };
+    VideoFrameRotation m_deviceOrientation { };
```

Comment:
This relies on `None` being 0. Maybe better to initialize this in the constructor in the cpp file?

Reply:
Good catch; I intended on adding the explicit initializer in the MockRealtimeVideoSource constructor, as a sort of belt-and-suspenders approach. Will fix.

Reply:
Also, in a previous review, Darin felt strongly that an enum ivar should always be initialized to _something_ in the header, even if it was then also initialized by a class constructor in the implementation file. So adding a initializer in the constructor wouldn't replace this zero-init here in the header, but supplement it.

Reply:
I still feel this way. The only way I would feel comfortable leaving out the `{ }` is if we could prove it’s a performance problem. And I am really glad that @heycam noticed we forgot to initialize to `None` in the constructor, which we should do. Finally, I think that “first value in enumeration is zero and is what you get with { }” is something that is OK to depend on, even though in this case I 100% agree with what @jernoble decided to do.